packages/docs/docs/zh/config/README.md

```diff
@@ -109,6 +109,24 @@ vuepress dev docs --no-cache     # 在每次构建前删除 cache
 ```
 :::
 
+### extraWatchFiles
+
+- 类型: `Array`
+- 默认值: `[]`
+
+指定额外的需要被监听的文件
+
+你可以监听任何想监听的文件，文件变动将会触发 `vuepress` 重新构建，并实时更新
+
+``` js
+module.exports = {
+  extraWatchFiles: [
+    '.vuepress/nav/en.js',
+    '.vuepress/nav/zh.js',
```

Comment:
One is the relative path and the other is the absolute path, both of which are best shown:

```js
module.exports = {
  extraWatchFiles: [
    '.vuepress/foo.js', // Relative path usage
    '/path/to/bar.js' // Absolute path usage
  ]
}
  
```